src/library/Uno.Themes.WinUI.Markup/Theme/ProgressRing.cs

```diff
@@ -0,0 +1,36 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Media;
+using Uno.Extensions.Markup.Internals;
+
+namespace Uno.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class ProgressRing
+		{
+			public static class Resources
+			{
+				public static class Default
```

Comment:
We should not have a `Default` class here

Right @agneszitte ?

Reply:
removed